mobile/src/main/java/org/openhab/habdroid/util/Util.java

```diff
@@ -99,7 +99,9 @@ public int compare(OpenHABSitemap sitemap1, OpenHABSitemap sitemap2) {
             try {
                 JSONObject sitemapJson = jsonArray.getJSONObject(i);
                 OpenHABSitemap openHABSitemap = new OpenHAB2Sitemap(sitemapJson);
-                sitemapList.add(openHABSitemap);
+                if(! (openHABSitemap.getName().equals("_default") && jsonArray.length() != 1)) {
```

Comment:
This will only work, if the _default sitemap is the last sitemap in the list. Is this always the case?

Reply:
Does it? jsonArray's size is always the same, doesnt matter at which position _default is.

Reply:
Damn, I haven't read jsonArray, I read sitemapList :( Sorry!